index.js

```diff
@@ -34,40 +34,18 @@ function runCodesign(src, opts, fn) {
   sign({
     app: src,
     hardenedRuntime: true,
-    identity: opts.identity
+    identity: opts.identity,
+    'gatekeeper-assess': false
```

Comment:
I’m not sure about this one. I needed it to let `codesign` finish locally, and Apple still said that notarization worked, so … probably not the end of the world to include this, especially if it makes things work?